eng/InstallRuntimes.proj

```diff
@@ -113,13 +113,14 @@
           Condition="$([MSBuild]::IsOsPlatform(Windows))"
           Inputs="$(VersionsPropsPath)" Outputs="$(TestConfigFileName)">
 
-    <Exec Command="$(PowershellWrapper) &quot;&amp; { &amp;$(DotnetInstallScriptCmd) $(CommonInstallArgs) -Version $(MicrosoftDotnetSdkInternalVersion) }&quot;" />
+    <Exec Command="$(PowershellWrapper) &quot;&amp; { &amp;$(DotnetInstallScriptCmd) $(CommonInstallArgs) -Version $(MicrosoftDotnetSdkInternalVersion) }&quot;"
```

Comment:
I'm going to take these changes to InstallRuntimes.proj and put up a separate PR to workaround the dotnet-install.ps1 issue so we can unblock all of our DARC updates.

Reply:
Sounds good. :)